bin/load-singlesshagent.sh

```diff
@@ -0,0 +1,67 @@
+# Make sure you source this script rather than executing it
+if [[ "${BASH_SOURCE[0]}" == "${0}" ]]
+then
```

Comment:
```suggestion
# This script ensures that the environment variables for the SSH agent are reused by all shells (including the AiiDA daemon)
# See https://aiida.readthedocs.io/projects/aiida-core/en/v1.5.0/howto/ssh.html
# Make sure to source this script rather than executing it
if [[ "${BASH_SOURCE[0]}" == "${0}" ]]
then
```